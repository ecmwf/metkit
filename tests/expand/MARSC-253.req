retrieve,
    class=od,
    type=tf,
    stream=enfo,
    expver=0001,
    repres=bu,
    obstype=32,
    date=20060112,
    time=0000/1200,
    domain=g,
    duplicates=keep,
    target=data.reference 