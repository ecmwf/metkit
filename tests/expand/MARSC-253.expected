RETRIEVE,
    CLASS      = OD,
    TYPE       = TF,
    STREAM     = ENFO,
    EXPVER     = 0001,
    REPRES     = BU,
    OBSTYPE    = 32,
    DATE       = 20060112,
    TIME       = 0000/1200,
    STEP       = 0,
    DOMAIN     = G,
    TARGET     = data.reference,
    DUPLICATES = KEEP 